02-express-tutorial/answers/final/006-basic-json.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const app = express();
+const {products} = require('./data');
```

Comment:
By default, paths given to `require` are relative to the location of the current file. In this case, the `data.json` file lives in the parent directory so you need a double-period to successfully run this file => `require('../data')`.

Reply:
Agreed. Same issues called out in the previous comments. Corrected now